core/src/main/resources/org/apache/spark/ui/static/sorttable.js

```diff
@@ -185,6 +185,10 @@ sorttable = {
         if (text.match(/^-?[£$¤]?[\d,.]+%?$/)) {
           return sorttable.sort_numeric;
         }
+        // match content like '287.1 MiB (17.2 GiB Remaining)', '0.0 B'
+        else if(text.match(/^(\d+(?:\.\d+)?)\s*(b|[k|m|g|t|p|e|z|y]ib)[\s\S]*/i)) {
```

Comment:
`else` keyword should be immediately after `if` block with one whitespace inserted between `}` and `else` like:
```
if ( ... ) {
} else if ( ... ) {
  ...
}
```

Also, you should insert one whitespace between `if` and `(`.